src/main/java/com/sonyericsson/jenkins/plugins/bfa/BuildFailureScanner.java

```diff
@@ -279,49 +279,70 @@ public static void scan(Run build, PrintStream scanLog) {
      * @param buildNum - Build object
      * @param buildUrl - Full URL of build
      * @param scanLog - PrintStream for the build log
-     * @return boolean true if message successfully created, false otherwise
+     * @return String Slack message with failure name, category and description if message successfully created, null otherwise
      */
-    public static boolean createSlackMessage(List<FoundFailureCause> foundCauseList,
+    public static String createSlackMessage(List<FoundFailureCause> foundCauseList,
```

Comment:
Normally changing a public signature like this is bad. But we haven't released a version with this method in yet, so it's fine. Just a reminder I might not approve such a change in the future. ;)

Reply:
Where is the change that handles the return value?

Reply:
The change is only in tests, since the value was never used.
By having the value as a String I could unit test against the value returned vs just true or false.